dot/digest/README.md

```diff
@@ -0,0 +1,94 @@
+[`Service` interface](https://pkg.go.dev/github.com/ChainSafe/gossamer/lib/services#Service), which means that it
+exposes `Start` and `Stop` functions. When a
+[Gossamer `Node`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot#Node) is started, a digest `Handler` is created
+with the [`NewHandler`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#NewHandler) function and started
+along with the Gossamer node's other services. The digest `Handler` maintains two `goroutines`, `handleBlockImport` and
+`handleBlockFinalisation`, which handle imported and finalised blocks respectively - these `goroutines` listen for
+messages on channels provided by the
+[`BlockState`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/state#BlockState).
+
+The digest `Handler` also exposes two public functions:
+[`NextGrandpaAuthorityChange`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#Handler.NextGrandpaAuthorityChange)
+and [`HandleDigests`](https://pkg.go.dev/github.com/ChainSafe/gossamer/dot/digest#Handler.HandleDigests).
+`NextGrandpaAuthorityChange` is consumed by the
+[GRANDPA service](https://pkg.go.dev/github.com/ChainSafe/gossamer/lib/grandpa#Service) to determine the block number of
+the next GRANDPA authority change.
+
+`HandleDigests` is invoked by the core service for each new block; this function reads the consensus digests from the
+block and dispatches actions based on the digest contents. The digest `Handler` supports digests from the BABE
+(authorship) and GRANDPA (finalisation) consensus engines - the following sections describe the messages in these
+digests and the actions Gossamer takes when it receives them.
+
+## BABE Messages
+
+[BABE](https://wiki.polkadot.network/docs/learn-consensus#block-production-babe) is a block production algorithm that
+assigns block authorship rights over periods of time referred to as "slots", which are a subdivision of an "epoch". One
+of the benefits of BABE is that it is able to provide a blockchain runtime with secure, decentralized randomness.
+Furthermore, BABE seeks to ensure the liveness of a blockchain by defining multiple tiers of potential block producers
+for each epoch. BABE digests may contain 1 of 3 messages, each of which are described below.
+
+### Next Epoch
+
+This message is issued by the runtime on the first block of every epoch - it provides the BABE consensus engine with the
+authority set and randomness for the _next_ epoch.
+
+### Disabled
+
+A message of this type will contain the ID of an authority; this authority should cease all authority functionality and
+all other authorities should ignore any authority-related messages from the identified authority. _Note: Gossamer has
+not implemented supported for messages of this type._
+
+### Next Config
+
+Messages of this type may only be issued in the first block of an epoch. These types of messages supply configuration
```

Comment:
`may` or `are`? 🤔

Reply:
In my mind "may" implies the specification and "are"/"does" implies a correct implementation.